### DISCUSSION TITLE
WMS Maßstabsbegrenzung CMS

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#170] [General]
Hi Jürgen,

ich glaube, dass ich einen Bug gefunden habe und zwar, wenn man bei einem WMS im CMS eine Maßstabsbegrenzung einstellt, dann wird das in der Darstellungsvariante nicht ausgegraut dargestellt. 
Einstellung CMS:
![grafik](https://github.com/gis-eni/webgis-forum/assets/119412172/d79b9c21-9c87-44c8-925a-770570f9ad6f)

Und dann die Darstellung im Webgis (Maßstab < 1:100.001). Die Daten werden korrekterweise nicht geladen, aber nicht ausgegraut dargestellt:
![grafik](https://github.com/gis-eni/webgis-forum/assets/119412172/74e65413-2905-4604-bb16-1744b03a447c)

**Comment by gis-eni** [comment]
Das Feature ist noch so ein Relikt von früher (WebGIS 3/4). Eigentlich sollte man Maßstäbe nicht im CMS definieren. Mittlerweile kann man die Grenzen im Dienst festlegen. Das ist auch stabiler und sicherer.

**Comment by thomaskofler** [comment]
Ok, der WMS ist ein externer Dienst, da habe ich ja keinen Einfluss auf die Maßstabsgrenzen oder wie meinst das "im Dienst festlegen"?. Die Daten von dem Dienst (HIS3D Profile) brauchen recht lange zu laden und damit kein Runtime-Error kommt, möchte ich halt eine Maßstabbegrenzung einfügen.
